eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -0,0 +1,103 @@
+package clusterloader2
+
+import (
+	//"encoding/json"
+	"fmt"
+	"github.com/aws/aws-k8s-tester/eksconfig"
+	k8sclient "github.com/aws/aws-k8s-tester/pkg/k8s-client"
+	gotemplate "github.com/aws/aws-k8s-tester/pkg/util"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"io/ioutil"
+	"net/http"
+	"path"
+	"strings"
+)
+
+type ClusterLoader struct {
+	Config    *eksconfig.Config
+	K8sClient k8sclient.EKS
+}
+
+// IsEnabled returns true if enabled
+func (c *ClusterLoader) IsEnabled() bool {
+	return c.Config.Spec.ClusterLoader != nil
+}
+
+// Apply installs the addon
+func (c *ClusterLoader) Apply() (err error) {
+	configMapData, err := c.buildConfigMapData()
+	if err != nil {
+		return fmt.Errorf("while forming configmap: %v", err)
+	}
+	template, err := gotemplate.FromLocalDirectory(struct {
+		*eksconfig.ClusterLoaderSpec
+		ConfigMapData map[string]string
+		TestArgs      string
+	}{
+		ClusterLoaderSpec: c.Config.Spec.ClusterLoader,
+		ConfigMapData:     configMapData,
+		TestArgs:          c.buildArgs(),
+	})
+	if err != nil {
+		return fmt.Errorf("while building templates, %v", err)
+	}
+	// Get Job info from goTemplate. Currently hardcoded.
+	resources := strings.Split(template.String(), "\n---\n")
+	jobYaml := resources[len(resources)-1]
+	if err := c.K8sClient.Delete(jobYaml); err != nil {
+		//Warn that there was an error, but don't stop process.
+		logger := zap.NewNop()
+		logger.Warn("Deleting jobYaml errored out, but is fine: %s", zapcore.Field{String: err.Error()})
```

Comment:
You don't want to make a new logger. That'll override settings we setup here:
https://github.com/aws/aws-k8s-tester/blob/master/pkg/logutil/zap.go

Try zap.S().Warn(...)